inst/config/text.yml

```diff
@@ -3,42 +3,50 @@ default:
   welcome:
     header: "Welcome to *tilt*'s online portal"
     body: >
-      *tilt* is a database designed to provide detailed insights into firms,
-      primarily focusing on small and medium enterprises (SMEs). It offers
-      extensive information on their climate profiles and transition risks
-      associated with evolving environmental standards. Recognising that the
-      creation of climate data involves a series of crucial decisions that
-      significantly influence outcomes, *tilt* aims to empower its users to
-      independently take these decisions. It allows them to thoughtfully consider
-      these decisions and customise methodological choices to best suit their
-      specific needs.
-
-  disclaimer:
-    header: "Disclaimer and license"
+    Our mission is to empower Small and Medium Enterprises (SMEs) across the EU and 
+    beyond with the funding and expertise needed to thrive in a net-zero future by 2050. 
+   
+    tilt  (Transforming in a Low Carbon Transition) is our data analytics solution 
+    designed to provide financial institutions with critical climate data, enabling them 
+    to offer SMEs the financing and knowledge essential for green transformation. 
+    By bridging the SME data gap, tilt reduces costs for financial institutions 
+    while supporting SMEs on their sustainability journey.
+    
+    Our goal is to drive investment in climate mitigation and adaptation by equipping 
+    financial institutions with the insights needed to provide SMEs with targeted financing.
+    
+    Watch this video to learn more about tilt. 
+    
+    header: "What You’ll Find on This Website"
     body: >
```

Comment:
@maurolepore could you please tell me what you mean with field? Maybe for context, I wanted to have more headings at one page. Is this not possible? I thought it is. Please find underneath the structure how I thought about it. 

The structure should be as such. We have a 
- WELCOMING PAGE (new body that is divided into sub-headings)
- DATA (no changes)
- DICTIONARY (with new body)
- METHODOLOGY (with new body)
- CLIMATE ACTION GUIDE (totally new tab, see ticket here: https://github.com/2DegreesInvesting/tiltWebTool/issues/196)

Here is the structure with the text - I put in bold if it is a heading and in capital if it is a new tab. Please also note that I put the links in here. 

WELCOMING PAGE

**Welcome to tilt's Online Portal**

Our mission is to empower Small and Medium Enterprises (SMEs) across the EU and beyond with the funding and expertise needed to thrive in a net-zero future by 2050.

_tilt_  (Transforming in a Low Carbon Transition) is our data analytics solution designed to provide financial institutions with critical climate data, enabling them to offer SMEs the financing and knowledge essential for green transformation. By bridging the SME data gap, tilt reduces costs for financial institutions while supporting SMEs on their sustainability journey.

Our goal is to drive investment in climate mitigation and adaptation by equipping financial institutions with the insights needed to provide SMEs with targeted financing.

Watch [this video](https://youtu.be/5lTpwTqgj_0) to learn more about tilt. 

**What You’ll Find on This User Platform**

- tilt’s Methodology: Explore our approach through training videos. Our methodology links minimal data inputs from SMEs to climate data (e.g., climate scenarios, Life-Cycle-Assessment data), generating climate indicators that reveal a company's emission intensity and climate risks.
- Research Data: Access research data available for non-commercial use (embedded link with license).
- Climate Action Guide: Discover our climate action guide, which standardizes the collection of revenue and climate action information from SMEs, improving data quality and enabling tailored sustainability loans.

These tools were developed under the CB-PASTAX program, funded by the EU. We've tested our product with five EU banks, with input from 21 banks in total. For applications beyond research, please visit www.tiltSMEs.org. Our prototype is launching with full-coverage in the Netherlands soon.

**We Value Your Feedback**

We strive for transparency and welcome input from financial institutions, researchers, and NGOs. Join the tilt community and share your [feedback here](https://docs.google.com/forms/d/1cwPCtlW3xP97aNIZ9Isge2zw5nJ-JnXvteckCO4oawo/viewform?edit_requested=true). 


METHODOLOGY
This paper introduces a methodology to derive climate metrics for SMEs,We present two indicators: a “relative emission indicator,” based on product emission intensity, and a “transition risk indicator,” assessing risks from the low-carbon transition. These indicators, built using Life Cycle Assessment (LCA) and climate scenario data, are tested on a database of 250,000 firms across five European countries.
Our method is reproducible, transparent, and scalable, requiring minimal data. It can be used to explore SME low-carbon transformation, climate risks, or inform financial decision-making.
Please find our [methodology here](https://docs.google.com/document/d/14syERZ-t1nzZaiotL77zcvKVqoJ_47NA/edit?usp=gmail_thread&rtpof=true&sd=true&userstoinvite=mauro%402degrees-investing.org&sharingaction=manageaccess&role=writer&ts=66c39331). For further information and how to use the research data, please watch these two [online webinars](https://youtu.be/WXNCYmXq7nw).


DICTIONARY
 The data dictionary provides detailed descriptions of the _tilt_ research
      data, including definitions, formats, and relationships for each data
      element. It serves as a guide to help you understand and utilize the data
      accurately and consistently within your analysis or applications.

Reply:
So to be clear so what is missing is the text

header: "What You’ll Find on This Website"
  body: >
tilt’s Methodology: Explore our approach through training videos. Our methodology links minimal data inputs from SMEs to climate data (e.g., climate scenarios, Life-Cycle-Assessment data), generating climate indicators that reveal a company's emission intensity and climate risks.
Research Data: Access research data available for non-commercial use (embedded link with license).
Climate Action Guide: Discover our climate action guide, which standardizes the collection of revenue and climate action information from SMEs, improving data quality and enabling tailored sustainability loans.
These tools were developed under the CB-PASTAX program, funded by the EU. We've tested our product with five EU banks, with input from 21 banks in total. For applications beyond research, please visit [www.tiltSMEs.org](http://www.tiltsmes.org/). Our prototype is launching with full-coverage in the Netherlands soon.